Docs/source/install/hpc/greatlakes.rst

```diff
@@ -0,0 +1,240 @@
+The cluster has various partitions, including `GPU nodes and CPU nodes <https://arc.umich.edu/greatlakes/configuration/>`__.
+
+
+Introduction
+------------
+
+If you are new to this system, **please see the following resources**:
+
+* `Great Lakes user guide <https://arc.umich.edu/greatlakes/>`__
+* Batch system: `Slurm <https://arc.umich.edu/greatlakes/slurm-user-guide/>`__
+* `Jupyter service <https://greatlakes.arc-ts.umich.edu>`__ (`documentation <https://arc.umich.edu/greatlakes/user-guide/#document-2>`__)
+* `Filesystems <https://arc.umich.edu/greatlakes/user-guide/#document-1>`__:
+
+  * ``$HOME``: per-user directory, use only for inputs, source and scripts; backed up (80GB)
+  * ``/scratch``: per-project `production directory <https://arc.umich.edu/greatlakes/user-guide/#scratchpolicies>`__; very fast for parallel jobs; purged every 60 days (10TB default)
+
+
+.. _building-greatlakes-preparation:
+
+Preparation
+-----------
+
+Use the following commands to download the WarpX source code:
+
+.. code-block:: bash
+
+   git clone https://github.com/ECP-WarpX/WarpX.git $HOME/src/warpx
+
+On Great Lakes, you can run either on GPU nodes with `fast V100 GPUs (recommended), the even faster A100 GPUs (only a few available) or CPU nodes <https://arc.umich.edu/greatlakes/configuration/>`__.
+
+.. tab-set::
+
+   .. tab-item:: V100 GPUs
+
+      We use system software modules, add environment hints and further dependencies via the file ``$HOME/greatlakes_v100_warpx.profile``.
+      Create it now:
+
+      .. code-block:: bash
+
+         cp $HOME/src/warpx/Tools/machines/greatlakes-umich/greatlakes_v100_warpx.profile.example $HOME/greatlakes_v100_warpx.profile
+
+      .. dropdown:: Script Details
+         :color: light
+         :icon: info
+         :animate: fade-in-slide-down
+
+         .. literalinclude:: ../../../../Tools/machines/greatlakes-umich/greatlakes_v100_warpx.profile.example
+            :language: bash
+
+      Edit the 2nd line of this script, which sets the ``export proj=""`` variable.
+      For example, if you are member of the project ``iloveplasma``, then run ``nano $HOME/greatlakes_v100_warpx.profile`` and edit line 2 to read:
+
+      .. code-block:: bash
+
+         export proj="iloveplasma"
+
+      Exit the ``nano`` editor with ``Ctrl`` + ``O`` (save) and then ``Ctrl`` + ``X`` (exit).
+
+      .. important::
+
+         Now, and as the first step on future logins to Great Lakes, activate these environment settings:
+
+         .. code-block:: bash
+
+            source $HOME/greatlakes_v100_warpx.profile
+
+      Finally, since Great Lakes does not yet provide software modules for some of our dependencies, install them once:
+
+      .. code-block:: bash
+
+         bash $HOME/src/warpx/Tools/machines/greatlakes-umich/install_v100_dependencies.sh
+         source ${HOME}/sw/greatlakes/v100/venvs/warpx-v100/bin/activate
```

Comment:
Above the guide informs the user to always `source $HOME/greatlakes_v100_warpx.profile`. 

Is activate copied to the greatlakes_v100_warpx.profile or are we loading a different source here? If so, why?

Reply:
This extra line is only needed initially, as we set up the dependencies.
The reason for that is that we already sourced the profile but only the line now adds the venv.